api/src/test/java/org/openmrs/module/metadatadeploy/handler/impl/RoleDeployHandlerTest.java

```diff
@@ -152,14 +152,14 @@ public void integration_shouldNotLoseInheritedRoles() throws Exception {
 	 * 
 	 * @return the current hibernate session.
 	 */
-	private org.hibernate.Session getCurrentSession() {
+	private org.openmrs.api.db.hibernate.DbSession getCurrentSession() {
```

Comment:
This method was added as apart of the original commit which should be reverted.

Reply:
when you talk of  reverting,you said it means the red and green which i see clearly there and even in the five files that i  changed for this pr